.travis.yml

```diff
@@ -2,8 +2,6 @@ jobs:
   include:
   - dist: xenial
     os: linux
-  - dist: trusty 
```

Comment:
If removing build testing on trusty i.e. ubuntu 14.04, we should add build testing for focal i.e. Ubuntu 20.04 https://docs.travis-ci.com/user/reference/linux/